confluent_kafka/src/confluent_kafka.c

```diff
@@ -327,13 +327,18 @@ static PyObject *Message_offset (Message *self, PyObject *ignore) {
 
 
 static PyObject *Message_timestamp (Message *self, PyObject *ignore) {
```

Comment:
I think it would be better to have a single timestamp() method that returns a tuple: tstype, timestamp